espoirMur/Andela-SendIT
# Ft user able login 162234065

#### What does this PR do?
Front end integration of login feature
#### Description of Task to be completed?
Consume api for login and enable login trought the front end 
#### How should this be manually tested?
By using correct login credentials to login or incorrect login credentials to check if a correct message is returned
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
